<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterConv2</class>
 <widget class="QMainWindow" name="FilterConv2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>568</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="typeCbb">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Mean</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gauss</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Median</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="sizeCbb">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="fiBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QLabel" name="orImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>261</height>
     </size>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="fiImage">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>451</width>
      <height>321</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="V_H">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical_Horizontal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sharp">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Sharpen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
